### ISSUE TITLE
Carousel sequence indicators have no visual affordance when focused

### ISSUE THREAD

**Issue by chrisdholt** [microsoft/fast-dna#2682]
**Describe the bug; what happened?**
Carousel sequence indicators have no visual affordance when focused. This is an accessibility violation and fairly severe, especially when autoplay is enabled. While there is a visual indicator for which slide you are on, there is no visual indicator for when you are focused on the control itself.

**What are the steps to reproduce the issue?**
1. Go to https://explore.fast.design/components/carousel
2. Navigate through the view using the keyboard
3. Notice there is no VFI for sequence indicators

**What behavior did you expect?**
A VFI exists and is visible when sequence indicator has focus

**In what environment did you see the issue?**
All Environments

**Comment by bheston** [comment]
Updated the design to include typical focus indicator. Note that spacing increased between items to allow for this.
![image](https://user-images.githubusercontent.com/47367562/74853599-00130b00-52f3-11ea-8c9b-66c25a8509af.png)
